src/DotVVM.Framework/Controls/HtmlGenericControl.cs

```diff
@@ -194,6 +200,31 @@ private void AddCssClassesToRender(IHtmlWriter writer)
             if (cssClassBindingGroup != null) writer.AddKnockoutDataBind("css", cssClassBindingGroup);
         }
 
+        private void AddStylePropertiesToRender(IHtmlWriter writer)
+        {
+            KnockoutBindingGroup cssStylesBindingGroup = null;
+            foreach (var styleProperty in CssStyles.Properties)
+            {
+                if (cssStylesBindingGroup == null) cssStylesBindingGroup = new KnockoutBindingGroup();
+                cssStylesBindingGroup.Add(styleProperty.GroupMemberName, this, styleProperty);
```

Comment:
This will add hard-coded values to binding group. I think it's not necessary because hard-coded values are added to `style` attribute and can't be changed.

Reply:
Good point. I changed it accordingly. thank you

Reply:
Ok, but it should be good enough to set the `nullBindingAction`. You don't need to call `HasValueBinding`.

Reply:
Or do you want to avoid rendering of empty binding group?

Reply:
I see. But yes, I wanted to avoid rendering `style` data-bind when there are only _hard-coded_ values. I didn't know about `nullBindingAction` and it's somehow strange that you have to allocate delegate to do nothing.

Reply:
Ok. Can you also change `CssClasses` to not render empty group in this case?

Reply:
I did. Is it okey?